lib/pid_ng/include/PID.hpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 202 COGIP Robotics association <cogip35@gmail.com>
+// This file is subject to the terms and conditions of the GNU Lesser
+// General Public License v2.1. See the file LICENSE in the top level
+// directory for more details.
+
+/// @ingroup     lib_pid
+/// @{
```

Comment:
Not closed

Reply:
done

Reply:
Dont resolve this one, I still have missing doxygen headers in other files

Reply:
done